frontend/src/containers/Restaurants/index.js

```diff
@@ -14,7 +14,7 @@ const mapDispatchToProps = {
     getRestaurants: restaurantsActions.startGetRestaurantsRequest
 };
 
-export default compose(
+export const RestaurantsContainer = compose(
```

Comment:
Try using "Restaurants" as the export name so you don't have to create another constant to export

Reply:
Done in [270a580](https://github.com/NickPriv/FryRank/pull/21/commits/270a5806712ecc2080dc3256cf5936d155029a8e).